specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/Jobs.json

```diff
@@ -430,9 +450,136 @@
         },
         "x-ms-examples": {
           "Terminate a Container Apps Job": {
-            "$ref": "./examples/Job_Stop.json"
+            "$ref": "./examples/Job_Stop_Execution.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
```

Comment:
We should also add the stopAll endpoint here and implement it to reduce overhead.

Reply:
@trajkobal can we reuse this endpoint? If the customer does not give a payload consisting of the job executions to be stopped, we stop all? I believe we would have to rename the endpoint to "StopAll" just so their is no confusion and the payload will make sense then, that if we have a payload passed we will only stop those job executions.?
OR 
we need a seperate endpoint /stopAll? The only concern with that is we would have 3 endpoints just for stop.

- /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/executions/{jobExecutionName}/stop
- /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stop
- /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stopAll

Reply:
What would be the difference between these two in this case?:

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stop
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stopAll

I think it is clearer to have only `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/stop`. If a the payload is empty, we stop all executions; and if the payload contains a list of executions, we stop only that ilst.